package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "road2",
-  "version": "2.0.0",
+  "version": "2.1.0-DEVELOP",
 
   "description": "Calcul d'itinéraire",
   "author": "RDEV - IGN",
   "main": "src/js/road2.js",
   "scripts": {
-    "start": "node ./src/js/road2.js",
+    "start": "env NODE_ENV=prod node ./src/js/road2.js",
```

Comment:
@jmkerloch is that change a problem for your dockerfile ? I thought I have deleted it...

Reply:
This is not an issue for me. We can change the command to whatever we want.